modules/products/bitbucket/variables.tf

```
@@ -0,0 +1,81 @@
+variable "environment_name" {
+  description = "Name of the environment."
+  type        = string
+  validation {
+    condition     = can(regex("^[a-zA-Z][a-zA-Z0-9\\-]{1,24}$", var.environment_name))
+    error_message = "Invalid environment name. Valid name is up to 25 characters starting with alphabet and followed by alphanumerics. '-' is allowed as well."
+  }
+}
+
+variable "namespace" {
+  description = "The namespace where Bitbucket pod will be installed."
+  type        = string
+}
+
+variable "vpc" {
+  description = "VPC module that hosts the products."
+  type        = any
+}
+
+variable "eks" {
+  description = "EKS module that hosts the product."
```

```suggestion
  description = "EKS module that hosts the products."
```

to align with `vpc` above (or we can use `product`)